src/_locales/ru/messages.json

```diff
@@ -169,26 +169,26 @@
     "html_suspended_modal_this_domain": { "message": "этот домен" },
     "html_suspended_modal_this_page": { "message": "эту страницу" },
     "html_suspended_modal_temp_whitelist": { "message": "Только эту вкладку в текущей сессии" },
-    "html_suspended_modal_temp_whitelist_notice": { "message": "вы всё ещё можете приостановить вручную." },
+    "html_suspended_modal_temp_whitelist_notice": { "message": "Вы все еще сможете приостановить вручную." },
     "html_suspended_modal_ok": { "message": "OK" },
     "html_suspended_modal_cancel": { "message": "Отмена" },
     "html_suspended_donation_question": { "message": "Вам понравилось это расширение?\nОтправьте пожертвование." },
     "html_permissions_title": { "message": "Разрешения" },
     "html_permissions_heading": { "message": "Пожалуйста, разрешите доступ к локальным файлам" },
-    "html_permissions_info": { "message": "У расширения в данный момент нет разрешения приостанавливать вкладки с URL (адресами), начинающимися со слова \"file\".\nЧтобы это изменить, включите \"Разрешить открывать локальные файлы по ссылкам\" со страницы настроек расширения." },
-    "html_permissions_backup": { "message": "Примечание: изменение этой опции приведет к временному исчезновению всех приостановленных вкладок.\nНа всякий случай рекомендуем сделать бэкап (резервную копию) текущей сессии вкладок." },
+    "html_permissions_info": { "message": "У расширения в данный момент нет разрешения приостанавливать вкладки с URL-адресами, начинающимися со слова \"file\".\nЧтобы это изменить, включите \"Разрешить открывать локальные файлы по ссылкам\" со страницы настроек расширения." },
+    "html_permissions_backup": { "message": "Примечание: изменение этой опции приведет к временному исчезновению всех приостановленных вкладок.\nНа всякий случай рекомендуем сделать бэкап (резервную копию) вкладок текущей сессии." },
```

Comment:
the word "backup" is made to mean manual copying or using session management? where can i see this message live? maybe you can give me a screenshot?

Reply:
I think it means both, at the user's discretion, user can just save the current session in session management or can export it to file and save externally, both can be done in session management, so no specification for backup word, but I guess we could add some explanation about where user can make a backup (coz users don't know anything), so it may look like: ..."сделать бэкап (резервную копию) вкладок текущей сессии в Управлении сессиями."

Reply:
okay, I corrected, only the preview above is "outdated"